Northstar.Client/mod.json

```diff
@@ -41,6 +41,10 @@
 			"Name": "modlist_reverse",
 			"DefaultValue": "0",
 			"Flags": 16777216
+		},
+		{
+			"Name": "ns_connected_server",
+			"DefaultValue": ""
 		}
```

Comment:
Weren't we going to make server name and stuff FCVAR_REPLICATED which would make this unnecessary?

Reply:
is there a pr for that?

Reply:
Don't think so, but it was talked about a lot in discord and stuff iirc?

Reply:
Wait they are? I thought that PR never got merged?